pkgs/development/python-modules/flake8-future-import/default.nix

```
@@ -20,7 +20,7 @@ buildPythonPackage rec {
   # Upstream disables this test case naturally on python 3, but it also fails
   # inside NixPkgs for python 2. Since it's going to be deleted, we just skip it
   # on py2 as well.
-  patches = lib.optionals isPy27 [ ./skip-test.patch ];
+  patches = lib.optionals isPy38 [ ./fix-annotations-version.patch ] ++ lib.optionals isPy27 [ ./skip-test.patch ];
```

getting kind of long
```suggestion
  patches = lib.optionals isPy38 [ ./fix-annotations-version.patch ]
    ++ lib.optionals isPy27 [ ./skip-test.patch ];
```